lib/fauna/client.rb

```diff
@@ -46,7 +46,7 @@ def initialize(params = {})
       @read_timeout = params[:read_timeout] || 60
       @connection_timeout = params[:connection_timeout] || 60
       @observer = params[:observer]
-      @adapter = params[:adapter] || Faraday.default_adapter
```

Comment:
Is there any sort of pooling configuration that needs to happen here, or is this implementation simply connection-per-thread?